src/pushcollector/_impl/proxy.py

```diff
@@ -50,11 +50,45 @@ class CollectorProxy(object):
     def __init__(self, delegate):
         self._delegate = delegate
 
+    def _translate_pushitem(self, pushitem):
+        if isinstance(pushitem, dict):
+            return [pushitem]
+
+        pushitems = []
+        checksums = {}
+        if pushitem.md5sum:
+            checksums["md5"] = pushitem.md5sum
+        if pushitem.sha256sum:
+            checksums["sha256"] = pushitem.sha256sum
+
+        push_item = {
+            "filename": pushitem.name,
+            "state": pushitem.state,
+            "src": pushitem.src,
+            "dest": None,
+            "checksums": checksums if checksums else None,
+            "origin": pushitem.origin,
+            "build": pushitem.build,
+            "signing_key": pushitem.signing_key,
+        }
+
+        if pushitem.dest:
+            for dest in pushitem.dest:
```

Comment:
This is just a stylistic preference, pointing it out in case you haven't considered it...

I'd have written this instead as:

```suggestion
        for dest in (pushitem.dest or []):
```